tests/api/api/test_projects.py

```diff
@@ -319,6 +319,80 @@ def test_list_and_get_project_summaries(
     )
 
 
+def test_list_project_summaries_different_installation_modes(
+    db: Session, client: TestClient, project_member_mode: str
+) -> None:
+    """
+    The list project summaries endpoint is used in our projects screen and tend to break in different installation modes
+    """
+    # create empty project
+    empty_project_name = "empty-project"
+    empty_project = mlrun.api.schemas.Project(
+        metadata=mlrun.api.schemas.ProjectMetadata(name=empty_project_name),
+    )
+    response = client.post("projects", json=empty_project.dict())
+    assert response.status_code == HTTPStatus.CREATED.value
+
+    mlrun.api.crud.Pipelines().list_pipelines = unittest.mock.Mock(
+        return_value=(0, None, [])
+    )
+    # Enterprise installation configuration post 3.4.0
+    mlrun.mlconf.igz_version = "3.6.0-b26.20210904121245"
+    mlrun.mlconf.kfp_url = "https://somekfp-url.com"
+    mlrun.mlconf.namespace = "default-tenant"
+
+    response = client.get("project-summaries")
+    assert response.status_code == HTTPStatus.OK.value
+    project_summaries_output = mlrun.api.schemas.ProjectSummariesOutput(
+        **response.json()
+    )
+    _assert_project_summary(
+        project_summaries_output.project_summaries[0], 0, 0, 0, 0, 0, 0, 0
```

Comment:
I understand that there is only one project in the list and that's why you are accessing the `0` index. I don't think the index should be set to a const, eventually in a case in which we will have 2 project in the tests we couldn't be 100% sure that the `empty project` will stay in the `0` index. In that case we will have to do a check inside the project summary name. In conclusion I think you should a comment for the reason why you are accessing the `0` index, what do you think?